src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -205,33 +217,41 @@ private AudioFileInfo merge(List<AudioFileInfo> audioFileInfos)
         return audioOperationService.merge(audioFileInfos, StringUtils.EMPTY);
     }
 
-    private AudioFileInfo trimToPeriod(AudioFileInfo audioFileInfo, OffsetDateTime trimPeriodStart,
-                                       OffsetDateTime trimPeriodEnd)
+    private AudioFileInfo trim(AudioFileInfo audioFileInfo,
+                               OffsetDateTime trimStartTime,
+                               OffsetDateTime trimEndTime)
         throws ExecutionException, InterruptedException, IOException {
-        var audioFileStartTime = audioFileInfo.getStartTime();
-
-        log.info("Trimming dates for ATS. Audio file start time is: " + audioFileStartTime.toString() + ", trim period start is: "
-                     + trimPeriodStart.toString() + ", trim period end is: " + trimPeriodEnd.toString());
-
-        var trimStartDuration = Duration.between(audioFileStartTime, trimPeriodStart);
-        var trimEndDuration = Duration.between(audioFileStartTime, trimPeriodEnd);
+        log.info("Trimming audioFileInfo [{}] from trimStartTime [{}] to trimEndTime [{}]",
+                 audioFileInfo, trimStartTime, trimEndTime
+        );
 
+        var audioFileStartTime = audioFileInfo.getStartTime();
         return audioOperationService.trim(
             StringUtils.EMPTY,
             audioFileInfo,
-            trimStartDuration,
-            trimEndDuration
+            Duration.between(audioFileStartTime, trimStartTime),
+            Duration.between(audioFileStartTime, trimEndTime)
         );
     }
 
 
-    private List<AudioFileInfo> trimAllToPeriod(List<AudioFileInfo> audioFileInfos, OffsetDateTime start,
-                                                OffsetDateTime end)
+    private List<AudioFileInfo> trimAll(List<AudioFileInfo> audioFileInfos,
+                                        OffsetDateTime mediaRequestStartTime,
+                                        OffsetDateTime mediaRequestEndTime)
         throws ExecutionException, InterruptedException, IOException {
         List<AudioFileInfo> processedAudios = new ArrayList<>();
         for (AudioFileInfo audioFileInfo : audioFileInfos) {
-            AudioFileInfo trimmedAudio = trimToPeriod(audioFileInfo, start, end);
-            processedAudios.add(trimmedAudio);
+            final boolean audioStartsBeforeRequest = audioFileInfo.getStartTime().isBefore(mediaRequestStartTime.toInstant());
+            final boolean audioEndsAfterRequest = audioFileInfo.getEndTime().isAfter(mediaRequestEndTime.toInstant());
+            if (audioStartsBeforeRequest || audioEndsAfterRequest) {
```

Comment:
I might be wrong. But if we are doing this check here, isn't then the below code we've introduced redundant:

```
if (numberOfSessions > 2) {
            finalisedAudioSessions.addAll(groupedAudioSessions.subList(1, numberOfSessions - 1)); // no need to trim mid session
        }
```

Reply:
I'd prefer that we steer away from doing any trim operation at all for the mid sessions as that may lead to erroneous files perhaps if it's understood that the audio files should not need to be touched by ffmpeg - they will already contain the ViqHeader for example, so I think they are best left untouched. It might also improve performance if there are a lot of mid session files to not need to process.